tests/storage-vm

```diff
@@ -500,7 +500,7 @@ for poolDriver in $poolDriverList; do
                 lxc delete v1 -f
                 lxc config set instances.migration.stateful=false
                 lxc init "${TEST_IMG:-ubuntu-minimal-daily:22.04}" v1 --vm -s "${poolName}"
-                lxc query /1.0/instances/v1 | jq -r '.expanded_config["migration.stateful"]' | grep -xF false
+                ! lxc query /1.0/instances/v1 | jq -r '.expanded_config["migration.stateful"]' >/dev/null # instances.migration.stateful leave it unset because since it is `false`, it is the same as the default value of `migration.stateful`.
```

Comment:
Can you do something like `lxc config get --expanded <instance> migration.stateful` and check the output is empty? Why using jq?

Reply:
fixed